app/js/components/visitsAndEncounters.jsx

```diff
@@ -188,16 +209,40 @@ export class VisitsAndEncounters extends React.Component {
       })
     return output
   }
+
+  onFiltersChange(value) {
+    let startValue = new Date(value.startDate.format('YYYY-MM-DD'));
+    let endValue = new Date(value.endDate.format('YYYY-MM-DD'));
+    let searchElements = document.getElementsByClassName("filterDate");
+    for(let i = 0; i < searchElements.length; i++) {
+        let colValue = searchElements[i].textContent;
+        let colDateValue= new Date(colValue);
+        let hide_content = searchElements[i].parentElement;
+        if ((colDateValue >= startValue ) && (colDateValue <= endValue)) {          
+          hide_content.style.visibility='visible';
+        }
+        else{
+          hide_content.style.visibility="hidden";
+        }
+  }
+  }
   
   render() {
+  let start = this.state.startDate.format('YYYY-MM-DD');
```

Comment:
Since `end` and `start` are not been re-assigned, you should use `const` instead

Reply:
@andela-dbamidele thanks for pointing out the indentation issue. I have fixed this. The end and start date do change everytime someone selects a new to and from dates.